i18n/en.json

```diff
@@ -2146,6 +2146,9 @@
   "admin.service.googleTitle": "Google API Key:",
   "admin.service.iconDescription": "When true, webhooks, slash commands and other integrations will be allowed to change the profile picture they post with. Note: Combined with allowing integrations to override usernames, users may be able to perform phishing attacks by attempting to impersonate other users.",
   "admin.service.iconTitle": "Enable integrations to override profile picture icons:",
+  "admin.service.InteractiveDialogTriggerTimeout.desc": "The number of milliseconds to wait for a response from the server when triggering an interactive dialog. If the server does not respond within the timeout, the dialog will not be displayed.",
```

Comment:
Nit: Let's keep the casing for these ids consistent with others in the admin definition, i.e. `interactiveDialogTriggerTimeout`

Reply:
done👍